src/leaflike/utils.clj

```diff
@@ -0,0 +1,21 @@
+(ns leaflike.utils
+  (:import [java.util Date TimeZone]
+           [java.text SimpleDateFormat]
+           [java.sql Timestamp]))
+
+(def email-pattern #"[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?")
+
+(def alpha-num-pattern #"^[a-zA-Z]+$")
+
+(defn get-timestamp
+  []
+  (let [date (Date.)]
+    (TimeZone/setDefault (TimeZone/getTimeZone "UTC"))
+    (.format (SimpleDateFormat. "yyyy-mm-dd hh:mm:ss") date)
+    (Timestamp. (.getTime date))))
+
+(defn required
```

Comment:
Looks like this is checking if a string is non-empty to short-circuit rest of the validation in users->validators. If that's the only use case for this method, this is redundant and can be removed